lib/tentacat/search.ex

```diff
@@ -11,13 +11,13 @@ defmodule Tentacat.Search do
   ## Example
 
       Tentacat.Search.code %{q: "code language:elixir repo:edgurgel/tentacat", sort: "url"}
-      Tentacat.Search.code %{q: "code language:elixir repo:edgurgel/tentacat", sort: "url"}, client
+      Tentacat.Search.code %{q: "code language:elixir repo:edgurgel/tentacat", sort: "url"}, client, [pagination: :none]
 
   More info at: https://developer.github.com/v3/search/#search-code
   """
-  @spec code(map, Client.t) :: Tentacat.response
-  def code(params, client \\ %Client{}) do
-    get "search/code", client, params
+  @spec code(map, Client.t, [atom]) :: Tentacat.response
```

Comment:
I think in this case `Keyword` instead of `[atom]` seems more accurate. Sorry for the super late reply